src/poliastro/plotting.py

```diff
@@ -82,8 +82,10 @@ def __init__(self, ax=None, num_points=150, dark=False):
             if dark:
                 with plt.style.context('dark_background'):
                     _, self.ax = plt.subplots(figsize=(6, 6))
+                    self.edgecolor = mpl.rcParams['figure.edgecolor']
```

Comment:
Could we do this without adding an extra property to `OrbitPlotter`?

Reply:
We have to do this because we are using temporary styling. And I could not get any property of the plot outside the ``OrbitPlotter`` itself. Do you have any idea? Because what we can get outside is a ``plt.subplots`` object which does not help :/

Reply:
You could also get `plt.gcf()` (Get Current Figure) and use `plt.gcf().get_facecolor()` there

Reply:
Yes! I did almost a similar thing!